src/mgr/QueryParams.cc

```diff
@@ -65,61 +69,69 @@ Mgr::QueryParams::find(const String& name) const
     return iter;
 }
 
-bool
-Mgr::QueryParams::ParseParam(const String& paramStr, Param& param)
+/// Parses the value part of a "param=value" URL section.
+/// Value can be a comma separated list of integers, or an opaque string.
+/// \note opaque string can be a list with any non-integer.
+///       For example; 1,2,3,4,5,z
+Mgr::QueryParam::Pointer
+ParseParamValue(SBuf &valueStr)
```

Comment:
Please keep the parameter constant and avoid `Str` suffix noise. I suggest:

```suggestion
ParseParamValue(const SBuf &rawValue)
```

I do not think this renaming increases the diff.